qa/tasks/cephfs/filesystem.py

```diff
@@ -1530,3 +1531,29 @@ def shrink(self, new_max_mds, status=None):
         assert(new_max_mds < oldmax)
         self.set_max_mds(new_max_mds)
         return self.wait_for_daemons()
+
+    def wait_until_scrub_complete(self, tag=None, rank=0, timeout=300):
+        # time out after "timeout" seconds and assume as done
+        with contextutil.safe_while(sleep=10, tries=timeout//10) as proceed:
```

Comment:
Add a `sleep` parameter. Revert `//10` to `//30` and pass `timeout=300` from `_wait_until_scrub_complete`. That would fix this code.

Reply:
This issue is fixed in the latest code. So resolving it.